docs/_pages/releases.md

```diff
@@ -21,6 +21,18 @@ sidebar:
 * Make `DefaultValueFormatter` and `EnumerableValueFormatter` suitable for inheritance - [#1295](https://github.com/fluentassertions/fluentassertions/pull/1295).
 * Added support for dictionary assertions on `IReadOnlyDictionary<TKey, TValue>` - [#1298](https://github.com/fluentassertions/fluentassertions/pull/1298).
 * `GenericAsyncFunctionAssertions` now has `AndWhichConstraint` overloads for `NotThrow[Async]` and `NotThrowAfter[Async]` - [#1289](https://github.com/fluentassertions/fluentassertions/pull/1289).
+* Added `ReturnTypes` to `MethodInfoSelector` to get all return types from all the methods selected
```

Comment:
This is a nice contribution, but I don't think it needs 12 lines of mentioning ;)
We could condense this some, without loosing information.
E.g. 
"Added `ThatAre[Not]Classes` and `ThatAre[Not]StaticClasses` to `TypeSelector`"

Reply:
Reduced :)